### PR TITLE
Fix for #620 and #224: prevent implicit conversion of nil to string

### PR DESCRIPTION
**Pull request by RobertSzkutak**
I implemented two fixes:

1) Properly setting ml.server
2) Changing v to v.to_s

In my testing, either fix appeared to independently solve #620 and #224

### CODE REVIEW

**Comment by grtjn** [comment]
I am actually thinking, why not just throw an error if ml.server isn't properly configured? Not sure if that will have side-effects on commands that don't need an env, but if it doesn't it would give proper feedback about properties being mal-configured..

**Comment by grtjn** [comment]
I'd still keep the second change by the way, just change the first into an exception..

**Comment by RobertSzkutak** [comment]
Pushed in a commit with your suggestion @grtjn

**Comment by grtjn** [comment]
Don't always blindly believe my suggestions.. ;-)

We should run some tests with this. I recall some feature in latest slush-marklogic-node depends on ./ml dev info returning results without dev.properties. Also not sure what happens with commands that don't take env (like ml upgrade and such)..

**Comment by RobertSzkutak** [comment]
I tried out every command and couldn't find any negative impact to upgrade, the general commands or the scaffolding commands. The bootstrapping commands and deployment/data commands app all appeared to be positively impacted. I learned that the settings command and some others threw nasty exceptions that were hard to parse as missing *.properties files.

I will investigate the impact on Slush soon.

**Comment by dmcassel** [comment]
Looks good to me. @grtjn you happy with this?

**Comment by RobertSzkutak** [comment]
@grtjn I did a review of Slush and can't find any code to suggest it would be impacted by this. I tried it out on a Slush install I already had up and running and it seemed ok. If Slush is impacted by this for some reason I'd personally prefer it finds another way to accomplish whatever it needs than by relying on returning non-existent properties..

**Comment by dmcassel** [comment]
This looks good to me.
